content/events/2020-portland-oregon/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Portland, Oregon
+date: '2020-09-12'
+location:
+  country: United States
+  city: Portland, Oregon
+  url: http://cnpdx.us/
+  venue: Puppet Offices
+map: 2020-portland-oregon-venue.png
+sched_slug: kcdpdx
+social:
+  twitter: 'https://twitter.com/cnpdx'
+organizers_email: info@cnpdx.us
+local_law_enforcement: 112
+medical_emergency: 112
+---
+
+Kubernetes Community Day Portland Oregon is the first of its kind in the city of Portland, Oregon.
```

Comment:
I kind of think instead of saying this putting something more clear (in case people drop on this page first rather than the KCD site first)
The Portland Kubernetes Community Day is a community-organized event that gathers people from open source and cloud native communities for education, collaboration, and networking. All knowledge levels are welcome.